pkg/metadata/labels.go

```diff
@@ -29,9 +30,8 @@ const (
 const (
 	NodeIDLabel = LabelKroPrefix + "node-id"
 
-	OwnedLabel           = LabelKroPrefix + "owned"
-	KroVersionLabel      = LabelKroPrefix + "kro-version"
-	ControllerPodIDLabel = LabelKroPrefix + "controller-pod-id"
+	OwnedLabel      = LabelKroPrefix + "owned"
+	KroVersionLabel = LabelKroPrefix + "kro-version"
```

Comment:
I'm being a super pedant here but:
https://google.github.io/styleguide/go/decisions#initialisms

`KRO` vs `Kro`.

Reply:
This applies elsewhere in this file too.

Reply:
I like it!! please keep them coming :)